docs/guides/server/bootstrap-admin-recovery.adoc

```diff
@@ -0,0 +1,90 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Admin bootstrap and recovery"
+summary="Learn how to bootstrap and recover admin account." >
+
+== A solution for a disaster scenario
+
+It used to be difficult to regain access to a {project_name} instance, where all admin users have been locked out. The process required multiple user-advanced steps, including direct database access and manual changes. In the continuous effort to improve the user experience, {project_name} now provides multiple ways to bootstrap a new admin account, which can be later used to recover from such a situation.
+
+== Bootstrapping an admin user using a dedicated command
+
+To create a temporary admin user, execute the following command:
+
+<@kc.bootstrapadmin parameters="user"/>
+
+This command can be executed even before the {project_name} first ever start. In other words, it does not matter, whether the `master` realm already exists or not. If no other parameters are specified and/or no corresponding environment variables are set, a user is prompted to enter required information. For more information, see the <<Environment variables>> section bellow.
+
+Alternatively, the parameters can be directly specified in the command:
+
+<@kc.bootstrapadmin parameters="user --username tmpadm --password tmpadmpass"/>
+
+This command creates a temporary admin user with the username `tmpadm` and the password `tmpadmpass`. The user is valid for 60 minutes.
```

Comment:
We do not automatically expire the user at this phase, see https://github.com/keycloak/keycloak/issues/30008#issuecomment-2151956072.
```suggestion
This command creates a temporary admin user with the username `tmpadm` and the password `tmpadmpass`.
```

Reply:
@vmuzikar you probably reviewed an older revision - at the time you posted the review the text didn't contain any reference regarding the expiration. I immediately cleaned it up after our discussion at the meeting.

Thanks for pointing it out in the fist place.

Reply:
This was a leftover I forgot about. Adressed, thanks!